code-challenges.test.js

```diff
@@ -11,40 +11,86 @@
+describe("duplicateAnswers", () => {
+    it ("takes in two arrays as arguments and returns one array with no duplicate values.", () => {
+        const dataArray1 = ["array", "object", "number", "string", "Boolean"]
+        const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
+        // Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
+        expect(duplicateAnswers(dataArray1,dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+        
+    })
+})
+// ReferenceError: duplicateAnswers is not defined
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 passed, 2 total
+
+// Create a function "duplicateAnswers", that takes in two parameters: 2 arrays
+const duplicateAnswers = (array1, array2) => {
+// We want to combine the two arrays without duplicate values, therefore, we use the spread operator with new Set to add the arrays to a new "set" object & 'set' will not allow for any duplicate values in an array only unique ones, so it's a win win situation.
+    return [...new Set ([...array1, ...array2])]
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
+
 
+// --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
+
+// a) Create a test with an expect statement using the variable provided. 
+// HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-// b) Create the function that makes the test pass.
+describe("shuffleArray", () => {
+    it ("takes in an array, removes the first item from the array and shuffles the remaining content.", () => {
+        const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+        // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+        const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+        // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+        // expect(shuffleArray(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+        // expect(shuffleArray(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+        // expect(["purple", "blue", "green", "yellow", "pink"]).toEqual(expect.shuffleArray(colors1))
+        // expect(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]).toEqual(expect.shuffleArray(colors2))
+        expect(["purple", "blue", "green", "yellow", "pink"]).not.toEqual(shuffleArray(colors1))
+        expect(["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]).not.toEqual(shuffleArray(colors2))
+    })
+})
+// ReferenceError: shuffleContent is not a function
+// OH MY LANTA THIS JEST TEST WAS SO HARD!!! I didn't even ignore the hyperlink, and somehow still struggled with the test. 
```

Comment:
This one is definitely tricky.  While the test is checking that there is some kind of modification to the array, it is not checking that there are certain values which is also important.  A combination of what you have and using the expect.arrayContaining could be really efficient in testing this problem.